t/MinecraftStatus.t

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+ddg_spice_test(
+    [qw( DDG::Spice::MinecraftStatus )],
+    'minecraft status' => test_spice(
+        '/js/spice/minecraft_status/status',
+        call_type => 'include',
+        caller => 'DDG::Spice::MinecraftStatus'
+    ),
+    'mc status' => test_spice(
+        '/js/spice/minecraft_status/status',
+        call_type => 'include',
+        caller => 'DDG::Spice::MinecraftStatus'
+    ),
+    'mcstatus' => undef;
```

Comment:
@destruc7i0n Please remove the semicolon here, it's causing your [tests to fail](https://travis-ci.org/duckduckgo/zeroclickinfo-spice/jobs/47955345#L1437).

We might want to add one more to trigger "is minecraft up?" to be on the safe side :)